website/source/docs/configuration/storage/s3.html.md

```diff
@@ -55,6 +55,9 @@ cause Vault to attempt to retrieve credentials from the AWS metadata service.
 - `session_token` `(string: "")` – Specifies the AWS session token. This can
   also be provided via the environment variable `AWS_SESSION_TOKEN`.
 
+- `max_parallel` `(int: 128)` – Specifies The maximum number of concurrent
+  requests to S3.
```

Comment:
Okay, easy enough to fix. Should I fix all the other ones that list it as an int as well? I copied this from the Azure docs, but Consul etc also list it as an int.